tests/util.test.ts

```diff
@@ -106,4 +106,27 @@ describe('Util', () => {
       expect(serializedError).toEqual(expected);
     });
   });
+
+  describe('redactSecret', () => {
+    test('secret without quotes', () => {
```

Comment:
It would be good to add some expectation what should happen, like "it should redact the secret"